### DISCUSSION TITLE
How to allow sd card access?

### DISCUSSION THREAD

**Discussion by pranataalam67** [deckerst/aves#474] [Q&A]
Show folders from sd card

**Comment by deckerst** [comment]
Aves shows media files as they are discovered by the OS. In other words, if Android discovered media on your SD card, Aves will display them.

Sometimes, especially on older devices, discovering media on SD cards can take time, or even require rebooting if you just inserted the card or transferred files to it.

Also, some SD cards are buggy and prevent the system from interacting with it normally.

Do other apps on your device show media on your SD card?
